bindings/node/index.d.ts

```diff
@@ -296,6 +296,25 @@ export interface ClientEncryptionCreateDataKeyProviderOptions {
    * If a key is created with alternate names, then encryption may refer to the key by the unique alternate name instead of by _id.
    */
   keyAltNames?: string[] | undefined;
+
+  /** @experimental */
+  keyMaterial?: Buffer | Binary;
+}
+
+/** @experimental */
+export interface ClientEncryptionRewrapManyDataKeyProviderOptions {
+  provider?: ClientEncryptionDataKeyProvider;
```

Comment:
The spec doesn't have this as optional - is there a case where it's not provided?